testcases/change_password_fail.robot

```diff
@@ -1,23 +1,26 @@
 *** Settings ***
-Library          DataDriver    file=../resources/change_pwd_fail_data.csv
-Resource         ../resources/import.resource
-Variables        ../resources/account_data.yaml
+Library    DataDriver    file=../test_data/change_pwd_fail_data.csv
+
+Resource    ../browser/browser.resource
+Resource    ../page_objects/header_navigation.resource
+Resource    ../page_objects/my_account_page.resource
+Resource    ../page_objects/change_password_page.resource
+Resource    ../keywords/custom_keywords.resource
 
 Test Template    Invalid change password Scenarios
 
-Suite Setup       Reach My Account Page After Logged In
+Suite Setup    Register New User Then Login
 Suite Teardown    Close Browser Window
 
 *** Test Cases ***
 Verify change password fail with invalid cred    ${testcase_id}  ${old_pwd}  ${new_pwd}  ${confirm_pwd}  
 ...  ${summary_error_message}  ${old_pwd_error_message}  ${new_pwd_error_message}  ${confirm_pwd_error_message}
-
 *** Keywords ***
 Invalid change password Scenarios
     [Arguments]    ${testcase_id}  ${old_pwd}  ${new_pwd}  ${confirm_pwd}  
     ...  ${summary_error_message}  ${old_pwd_error_message}  ${new_pwd_error_message}  ${confirm_pwd_error_message}
-
-    Click Change Password Link
+    Click On My Account Link
+    Click On Change Password Link
```

Comment:
you can use url to visit in this case. because you click on UI in positive case

Reply:
Used the url instead.